src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/AddDefaultBillingAddressOnOrderFormSubscriber.php

```diff
@@ -37,10 +37,24 @@ public function preSubmit(FormEvent $event)
     {
         $orderData = $event->getData();
 
-        if (!isset($orderData['differentBillingAddress']) || false === $orderData['differentBillingAddress']) {
+        if ($this->shouldBillingAndShippingAddressBeTheSame($orderData)) {
             $orderData['billingAddress'] = $orderData['shippingAddress'];
 
             $event->setData($orderData);
         }
     }
+
+    /**
+     * @param array $orderData
+     *
+     * @return bool
+     */
+    private function shouldBillingAndShippingAddressBeTheSame(array $orderData)
+    {
+        return
+            (!isset($orderData['differentBillingAddress']) ||
+            false === $orderData['differentBillingAddress']) &&
```

Comment:
Can we merge this two first conditions into one line? It would be a little bit cleaner.